### PR TITLE
Support m.room.tombstone events

### PR DESCRIPTION
**Pull request by dbkr**
For https://github.com/vector-im/riot-web/issues/7164

### CODE REVIEW

**Comment by dbkr** [comment]
Don't merge just yet - may have replacement strings

File: src/components/views/rooms/MessageComposer.js [hunk 1]
```diff
@@ -61,6 +64,7 @@ export default class MessageComposer extends React.Component {
             },
             showFormatting: SettingsStore.getValue('MessageComposer.showFormatting'),
             isQuoting: Boolean(RoomViewStore.getQuotingEvent()),
+            tombstone: this._getRoomTombstone(),
```
Comment (bwindels) [inline src/components/views/rooms/MessageComposer.js]: Could we put this in the parent of the composer (`RoomView` or a new in-between component)? Apart from where it appears on the screen, it's not really related to composing messages.

Comment (bwindels) [reply src/components/views/rooms/MessageComposer.js]: With "this", I mean the whole tombstone code and show-a-message-instead-of-composer logic

Comment (dbkr) [reply src/components/views/rooms/MessageComposer.js]: True, although actually MessageComposer is kind of already a (probably badly named) wrapper around MessageComposerInput which is the actual text field. We'd also have to split out the avatar & room encryption icon as we still want these displayed.

Comment (bwindels) [reply src/components/views/rooms/MessageComposer.js]: Hmm, ok, that does sound painful. Better leave it as it is then.

**Pull request merged**